src/Server/Storage/FileSystem.fs

```diff
@@ -12,10 +13,10 @@ let getWishListFromDB userName =
     if not fi.Exists then Defaults.defaultWishList userName
     else
         File.ReadAllText(fi.FullName)
-        |> FableJson.ofJson<WishList>
+        |> Decode.Auto.unsafeFromString<WishList>
 
 let saveWishListToDB wishList =
     let fi = FileInfo(getJSONFileName wishList.UserName)
     if not fi.Directory.Exists then
         fi.Directory.Create()
-    File.WriteAllText(fi.FullName, FableJson.toJson wishList)
\ No newline at end of file
+    File.WriteAllText(fi.FullName, Encode.Auto.toString(4, wishList))
```

Comment:
@MangelMaxime  what's the 4 supposed to mean here?

Reply:
It's the number of spaces used for indendation.

In general, I set it to `4` in DEBUG mode and `0` for production usage.